cv_next/app/actions/cvs/fetchCvs/route.ts

```diff
@@ -0,0 +1,22 @@
+"use server";
+
+import { filterValues } from "@/app/feed/components/filterPanel";
```

Comment:
If you're using it outside the filter panel component, please move it to a more general location